prowler/lib/check/models.py

```diff
@@ -158,6 +159,184 @@ def get_bulk(provider: str) -> dict[str, "CheckMetadata"]:
 
         return bulk_check_metadata
 
+    @staticmethod
+    def list(
+        bulk_checks_metadata: dict = None,
+        bulk_compliance_frameworks: dict = None,
```

Comment:
do we need to force the caller to pass these values?

Reply:
No, done